### DISCUSSION TITLE
Kubecon prize?

### DISCUSSION THREAD

**Discussion by ixodie** [orb-community/orb#392] [Q&A] [answered]
Hi.

i think I won a switch today!  A woman at your booth told me if I was #124 to star the orb repo that I would win a switch.  So I waited and was #124 when it hit but now it seems people have unstarred it and it is at 123.  Is there anything I can do?  The repo seems interesting and I already see two performance improvement opportunities in the code.

Anyhow, thank you for speaking with me today.

-J

**Comment by weyrick** [comment, marked as answer]
@ixodie so sorry for us missing this post! It looks like we have a problem with our notifications which we will correct.

Please send an email to events@ns1.com with your question about the switch. We're so glad you stopped by the booth! 

Please let me know If you're still interested in contributing any improvements to Orb, I'd be happy to discuss or to review a PR.

Thanks,
Shannon
